notebooks/3_SinglePlotDiagnostics/1_Ins_SingleModelVisualDiagnostics.ipynb

```diff
@@ -0,0 +1,827 @@
j1ywYxlsfROO/QJ4+1/ePx2UUvz8ksM5Yvwwvv7YSlbtTLGLi3DMezuauOK3S4hYmsduPE6SMyFE5hSUmykGQh3Jv0abwmFdc8gJIYZYgvbC96FtL1z8v2mfW0aPP8Z0oVz9BLxxm6Pr3lLXxs+eW8Oxt77Ct55YRThqceulh7Pk5tO56ZxZjC4vdOaNgm3w+GcgvwTO+5/0tmwtvN6Mq3r+O6ZKoTjU6z+H4uHpvbkwAAV+L/dfczRVJXlc+8d3WL934BPQitS8tr6WT92/jPJCP09+4XjmjElv670QQhykwK4Om0o3R2lBE+IQme3iGGiB3e+bfxdVQtU08DuUTPRl08vw/oNw4tdh7NFpf7vj/+tVGtrncpv3VC55/VZ+saSJFcMvZdqIEqaNKGHGyFJmjCxJeoLNXfs7WbalgSffq2HxpgZ8HsVZh43kU8dM5PipVc7PQRWNwF9vgPoNptthaZrHr3g8cNG98OsT4Llvwicecq+r4/6dZj9trwUUlIyAMfOhfKw78QBsegW2vA5n/zyrSzePKCvgz58/lit++zZX37+UP31uEXPHOlvOXXTvqfdr+PYTHzBzVCkPfHaRTDwthMi8fPumUKAZykYn9xqp4ijEITKboDVsgj/FjZ3x5sG4hTD1Y6ZkeLIHc6pa98LfvgzVM+CUm9PzHgk+c9wk9neGWB3+MbM23cxXW37LvfsL+O2O42gPHZjAsrI4j0lVRUyqKqa6NJ/SfB/F+T46w1H2tQTY2xzgw13N7G4OADB2WCHfPnsmVywYx4jSNJU0j4bhb1+E9f+A8+6AKaem530SVU+H074HL/8Q3n8I5l+TmfcFaN0HS+6Fdc9B4+Ye4psBM881RU0qJmYutmjYjM+rnJK1rWfxJlQV8eiNx/Gp+5dy9f1Luf3yIzhn7sDHpAXCUVZsb+KdrY1sqW9na30bNU2ddASjJo/O9zG+sogp1cVMG1HCsVOqOHL8sJyvKmlZmjtf2sC9r23iuClV3PeZoyktyK4usEKIIcKXZ268B1uSf420oAlxiMxOVB1shT0fmP7GHfWwawVsfcsUoFBemHU+nPQNGHNU/98jUbgT/nQR7F0N178MIw9zbt3JCnXAo5+ELa+hT/oWu4/6OhvqOti4r5Wt9R1sq29ne0M7jR2hrkmcAcoL/YwozWfGyFIWTqpg4eRKZo0qS+8FZ8seeOpGM9bp9B+av0cmWVF48BLYuQw+97yz+0J32mrhX3fB8t+bRGja6TDlNDOVQNkYM36wZTfsWAKbXzWtWGiYeZ6Z927MkemND+DVn5qqoFc/ahLEQWLX/k6+8OAKVu9q5sIjxnDT2TMZX5n85KVN7SEWb67ng5pmPtzVzIrtTQQjFh5lpi2YVF3MhMpCSvJNMtISCLOjoYMtdW1dNzTKC/2cOL2ai44Yw2mzHBiTmWU6QhG+8dgqnv9oL1ctHM+PL5qb+sTnDpGJqoUQgLmuiwRg+pnJLd+0DWqWw4xzzJAKIQYhp78DM5ug9aRhM6z4I7z3oJk/Y9YFpiVloMlUNAxPXGdaRa74Ixx2iSPh9kskBM993bQMTTwBLrzbtBglCEct2oMRCvze5Erit9XCtrdMQrVvjemW17kf/EVmKoHycVA6yvQLHz7TFEgZPhMKKw5ez/4dsPJhWPK/ZnLNC+6EIz/p0ManqL0e7jsVIkG4/iWomOT8e7TVweK74N3fQzQI866Ck7/V9+SazTXmNct/b7pwOLWv9mTTy/DQ5aaF+eJfpec90igctbj31U389s3NRKKaM2aP5IIjRrNwUiUjSvO7uuZ2hCLsaupkc10772xtZNnWBtbsaUFryPN6ulrETppezaLJlRTn9974v78jxL821fPG+jpeW19LfVuIquI8Lj5qLJcfPY7Zowf/2KzVNc1884mVbKpt4/vnz+FzJ0xyvqtzCiRBE0IA5kZ8w0aYc4kZvpAoGob2ugPFQdr2QeNWc5M+U8NehHBYbiZoMYEWWPpr09Us2GoSqlO/a+bKSlWwFZ74LGx6Cc69HY650fl4+2Plw/DPmyHUCnMvNy1Uqcz91dEI2xebhGzrW1C31jyeXwajj4CSkSb5igRMAtG0zbymsxHCcVWVSkaaxE15TBfQ1j3m8Znnw5k/dn8S5Lr18PuzTJJ57d9N9z4ntNXB23ebJCsSMHPWnXJT34lZokCzSWaX/q/Z1+ZeavbVbpLuftv9PjxwgUlQP/9iVo8968ve5gC/e2sLf31/F43tIQAK/B5KC/yEoxb7O8Jdy+b7PMyfUMExUyo5ecZw5o0txzeAlq9w1OLNDXX8ZUUNL6/dRziqmTu2jE8sGM/HjxxLeeHg6g4YS3rvfW0T1SV53H75EZw8Y7jbYUmCJoQwmrZDzbvmhrA3YZx9uN08b0UOflwpmH0ReIfe7E8iN+R2ghbT0WiStKW/MUnFnI+b4h7Jdnfb+S489W/QtBUuuAuOvja98aaqrRYW3w3L/2hOVsNnwfSzTHe5islQVAVoc3cpEoR9H5nm/+2LTVdNtGkhm3AsTD7Z/Iw6ovcTm2WZsv9166FunfndXmu68BVXw6jDTfc5pxIhJ+xeabo7Ko+Zg23ySf1fV+teWPIrePd3dmJ2hemiONCEqqMR3r4Hlv0WIp0w7xMm4Rvo51izHB66FPLLTXKWrvGZGRaOWny0u4WVO5rYtb+TtmAUrwdGlxcyrqKQCZVFzBlTRr4vPWMRmtpDPL1yF48tr2HtnhbyfR7OnTuKKxeO55jJVVk9Xk1rzesb6vivf6xj/b5WLjlqLLdceBjlRdmRYEqCJoQATAXojS/0PMVQ+XjzHRk/XYwvX1rPxKA2NBK0mLY600Lx7u/MgNMpp5km8OrpUD3TbgGyL6giQdj2L9NVcu0zZjLfy34Hk07IXLyp6miEVY/AhhdM8pV4RymerxDGLTDJ2KSTTCXKNE8VkBXqN8IjV5kCM3Mvg9P/X/JdHrWGvR+Y1rJVj5jPd+5lcPJN
```

Comment:
This example seems to be a little unfair with MH, because the implementation you are using does not use any tuning, and any modern MH implementation will do it. And in the next section youuse NUTS from PyMC3 that is a highly tuned MCMC. One solution will be to use MH from PyMC3, the other just mention that the used MH implementation is not using any auto-tuning.

Reply:
I added a section and activity on PyMC3 MH! Thanks for the feedback